### DISCUSSION TITLE
Problem with connection esp32 to a smartphone hotspot

### DISCUSSION THREAD

**Discussion by opanyki314** [Hieromon/AutoConnect#468] [Q&A]
Hi! I use "autoconnect" library and sometimes I need to connect my esp32 to a smartphone hotspot. And everything worked well until recently. Now I can't connect to a hotspot. Esp32 gets an IP address and I see "Connected devices: 1 device" for a moment and then connection has lost. I see "No connected devices" (android 11). Can it be because a wrong configuration autoconnect?

**Comment by Hieromon** [comment]
There are no functional changes related to connecting AutoConnect to mobile hotspots. If it worked before, it's a phenomenon caused by changes in your environment.
This topic will be moved to [Discussions](https://github.com/Hieromon/AutoConnect/discussions) once as it is not accompanied by [information for problem diagnosis](https://hieromon.github.io/AutoConnect/faq.html#4-reports-the-issue-to-autoconnect-github-repository) as an issue.
